_appmap/env.py

```diff
@@ -11,6 +12,20 @@
 
 from . import trace_logger
 
+_ENABLED_BY_DEFAULT_MSG = """
+
+The APPMAP environment variable is unset. Your code will be
+instrumented and recorded according to the configuration in appmap.yml.
+
+Starting with version 2, this behavior will change: when APPMAP is
+unset, no code will be instrumented. You will need to use the
+appmap-python script to run your application, or explicitly set
```

Comment:
Does this message show if they are already running the launcher script or have set APPMAP?